A couple of small fixes to scripts

sryabkov opened the pull request:
Credit to David_Dyer for noticing an issue with DeployTemplate.ps1. David left the following note at the bottom of the page:

```
David_Dyer Dec 2, 2017
DeployTemplate.ps1 - Line 35 - New-AzureStorageContext cmdlet is missing the mandatory StorageAccountName parameter
```

azurecla commented:
Hi __@jessebenson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jesseb). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

sryabkov committed:
A couple of small fixes to scripts

Credit to David_Dyer for noticing an issue with DeployTemplate.ps1. David left the following note at the bottom of the page:

```
David_Dyer Dec 2, 2017
DeployTemplate.ps1 - Line 35 - New-AzureStorageContext cmdlet is missing the mandatory StorageAccountName parameter
```
